superset-frontend/spec/javascripts/datasource/DatasourceEditor_spec.jsx

```diff
@@ -17,12 +17,12 @@
  * under the License.
  */
 import React from 'react';
-import { Tabs } from 'react-bootstrap';
 import { shallow } from 'enzyme';
 import configureStore from 'redux-mock-store';
 import fetchMock from 'fetch-mock';
 import thunk from 'redux-thunk';
 
+import Tabs from 'src/common/components/Tabs';
```

Comment:
Let's follow the same pattern as for `'src/components/Popover/index.tsx` where we have a placeholder file and all internal references point to that file.

Reply:
The same comment applies here as in the Popover PR - https://github.com/apache/incubator-superset/pull/11163#discussion_r501041041

Reply:
@kgabryje let us know if you plan to move `.../Tabs.tsx` to `.../Tabs/index.tsx` in this PR, or if it needs to be a follow-up.

Reply:
Done. Other components in `/common/components` should be refactored the same way, but I suggest we do that in a separate PR.
I tried to move stories from `common.stories.tsx` to `Tabs/tabs.stories.tsx`, but it did not work - they didn't appear in Storybook's nav menu. Is that a known bug? In `.storybook/main.js` there's a line `stories: ['../src/@(components|common)/**/*.stories.@(t|j)sx']` which seems to cover the cases of stories in sub dirs, but I think it doesn't work as expected.